docs/UserGuide.md

```diff
@@ -65,17 +65,14 @@ It is a convenient platform for you to keep track of your lessons and assignment
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds an assignment into your schedule.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
```

Comment:
Since we have some optional fields as well, maybe it would be clearer to the user if we changed the description given in AB3 to match our project?

Reply:
wait what do you mean? is it we allow them to have any number of tags as well?

Reply:
I was suggesting to keep the Tip section and change the description to "You can use the remind tag to set reminders for the assignment you want to add instead of using the remind command after adding the assignment" or something along those lines.

Reply:
Okay updated!